cortico.js

```diff
@@ -1500,6 +1535,19 @@ async function setupPreferredPharmacy(code, demographic_no) {
 }
 
 
+async function getDiagnosticFromCortico(appt_no, notes) {
+  const clinicName = localStorage['clinicname']
+  const url = `https://${clinicName}.cortico.ca/api/encrypted/diagnostic-results/?appointment_id=${appt_no}&notes=${notes}`
+
+  return fetch(url, {
+    method: "GET",
+    headers: {
```

Comment:
What if we're not logged in to Cortico on the clinic in question? It should indicate that, and post a link with target=_blank, to sign in, telling the user to "please log into Cortico and try again."